src/index.tsx

```diff
@@ -89,25 +91,27 @@ const EZDrawer: React.FC<Props> = function ({
     }
 
     return (
-        <div id='EZDrawer'>
+        <div id={'EZDrawer' + idSuffix} className='EZDrawer'>
```

Comment:
What is the usage of id in here?

As it's not used anywhere now.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id => `The id [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) defines an identifier (ID) which must be unique in the whole document.`